CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # splat Release Notes
 
+### 0.24.5
+
+* New yaml option: `global_vram_start` and `global_vram_end`.
+  * Allow specifying that the global segment may be larger than what was automatically detected.
```

Comment:
I'm not sure I like the terminology "the global segment" since many segments can be global. is it the first segment in global memory space, or any segment in global memory?

Reply:
or is this just a matter of changing "the" to "a"? maybe that's all we need